### PR TITLE
Added dmac functions

### PR DESCRIPTION
**Pull request by TheOfficialFloW**

### CODE REVIEW

**Comment by devnoname120** [comment]
@TheOfficialFloW Please fix. https://travis-ci.org/vitasdk/vita-headers/builds/232107134#L190-L193

File: include/psp2kern/kernel/dmac.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * \usergroup{SceKernelDmacMgr}
```
Comment (devnoname120) [inline include/psp2kern/kernel/dmac.h]: `\kernelgroup`

File: include/psp2kern/kernel/dmac.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/**
+ * \usergroup{SceKernelDmacMgr}
+ * \usage{psp2kern/kernel/dmac.h,SceKernelDmacMgr_stub}
```
Comment (devnoname120) [inline include/psp2kern/kernel/dmac.h]: `SceDmacmgrForDriver_stub`

File: include/psp2/kernel/dmac.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/**
+ * \usergroup{SceKernelDmacMgr}
+ * \usage{psp2/kernel/dmac.h,SceKernelDmacMgr_stub}
```
Comment (devnoname120) [inline include/psp2/kernel/dmac.h]: `SceDmacmgr_stub`

File: include/psp2kern/kernel/dmac.h (hunk 1 above)
Comment (devnoname120) [inline include/psp2kern/kernel/dmac.h]: Also, the `Kernel` is [usually omitted](https://github.com/vitasdk/vita-headers/tree/master/include/psp2/kernel). Not sure this is a good idea to omit it, but I guess it's better to stick with conventions, and maybe open an issue to discuss the matter.

File: include/psp2/kernel/dmac.h [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/**
+ * \usergroup{SceKernelDmacMgr}
```
Comment (devnoname120) [inline include/psp2/kernel/dmac.h]: Same comment here about the `Kernel` part.

**Comment by devnoname120** [comment]
You need to add the group here: https://github.com/vitasdk/vita-headers/blob/master/docs/definitions.dox

File: docs/definitions.dox [hunk 5]
```diff
@@ -134,6 +134,9 @@
  *     \defgroup SceSysmem System Memory Library
  *      Find/Get/Open memory blocks, get the model of the Vita
  *
+ *     \defgroup SceDmacMgr Direct Memory Access Controller
+ *      Direct Memcpy and Memset to private memory locations
```
Comment (devnoname120) [inline docs/definitions.dox]: @TheOfficialFloW Please confirm the description is correct.

Comment (TheOfficialFloW) [inline docs/definitions.dox]: Looks good thanks.

**Pull request merged**